### PR TITLE
[15.0][MIG] web_sheet_full_width: Migration to 15.0

### PR DESCRIPTION
**Pull request by BankyKmitl**

### CODE REVIEW

File: web_sheet_full_width/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,8 @@
+This module was written to extend the functionality of the web client
+to get full width in the form view sheet.
+
+This module works in community edition and in enterprise edition.
+
+This module feature and many other features and improvements to the web backend
+are included in `web_responsive module <https://github.com/OCA/web/tree/13.0/web_responsive>`_.
```
Comment (traviswaelbro) [inline web_sheet_full_width/readme/DESCRIPTION.rst]: Should this be updated to `15.0` even though OCA/web#2058 isn't merged yet?

```suggestion
are included in `web_responsive module <https://github.com/OCA/web/tree/15.0/web_responsive>`_.
```

**Review by traviswaelbro — COMMENTED** [review]
Tested and looks good for the form's full width sheet. See attached images for Community & Enterprise comparisons.

[Community with web_sheet_full_width](https://user-images.githubusercontent.com/7907616/145224461-0e8898cf-aaaa-4507-afbc-e06310c21bfb.png)
[Community without web_sheet_full_width](https://user-images.githubusercontent.com/7907616/145224455-756247d6-e488-4e0c-a71b-cf2d344ea1f9.png)
[Enterprise with web_sheet_full_width](https://user-images.githubusercontent.com/7907616/145224468-65282120-c3c1-4af5-8b65-28e1c71903aa.png)
[Enterprise without web_sheet_full_width](https://user-images.githubusercontent.com/7907616/145224473-dcc3d4f7-2a8e-4406-942d-d46f89cebe30.png)

File: web_sheet_full_width/readme/DESCRIPTION.rst (hunk 1 above)
Comment (BankyKmitl) [reply web_sheet_full_width/readme/DESCRIPTION.rst]: fixed, thank you.

**Comment by BankyKmitl** [comment]
@pedrobaeza Could you check this for a merge?

**Comment by matiasperalta1** [comment]
@BankyKmitl @pedrobaeza Could you check if this module is ready for merging?

**Comment by pedrobaeza** [comment]
Other PR was merged.

**Pull request closed without merging**